stock_picking_auto_create_lot_biochousa/models/product.py

```diff
@@ -6,4 +6,10 @@
 class ProductTemplate(models.Model):
     _inherit = "product.template"
 
-    exception_eco_partner_ids = fields.Many2many('res.partner', string="Exception Eco Productor")
\ No newline at end of file
+    exception_eco_partner_ids = fields.Many2many('res.partner', string="Exception Eco Productor")
+
+
+class PartnerTemplate(models.Model):
+    _inherit = "res.partner"
+
+    exception_eco_partner_ids = fields.Many2many('product.template', string="Exception Eco Productor")
```

Comment:
Some needed fixes here:
* The field should be named `exception_eco_product_ids` (not partner)
* Too long line ( > 80).
* Missing empty line